components/raftstore/src/store/worker/region.rs

```diff
@@ -887,10 +894,6 @@ where
             }
         }
     }
-
-    fn shutdown(&mut self) {
-        self.pool.shutdown();
```

Comment:
Do you needs to impl it?

Reply:
I think no place to use the shutdown current.